lms/templates/static_pdfbook.html

```diff
@@ -1,8 +1,14 @@
+<%page expression_filter="h"/>
 <%! from django.utils.translation import ugettext as _ %>
 
 <%inherit file="main.html" />
 <%namespace name='static' file='static_content.html'/>
-<%block name="pagetitle">${_('{course_number} Textbook').format(course_number=course.display_number_with_default) | h}</%block>
+<%!
+from openedx.core.djangolib.js_utils import (
```

Comment:
Yes, much like the above comment, in the event that JS fails, focus goes to the ID wrapper for the textbook.

But, since JS is required, the normal functionality we've defined takes over. This `href` just allows tests to pass by creating a focusable target.